Core/Src/spi_command_handler.c

```diff
@@ -227,16 +234,34 @@ void spi_transfer_image() {
     num_transfers =
         (uint16_t)((image_length + (IRIS_IMAGE_TRANSFER_BLOCK_SIZE - 1)) / IRIS_IMAGE_TRANSFER_BLOCK_SIZE);
 
-    spi_init_burst(VIS_SENSOR);
```

Comment:
Could we consider having this spi_transfer_image() function take in an integer (say, 0 for VIS and 1 for NIR) instead of having it manually defined? Then I suppose there could be a vis image transfer opcode and a nir transfer opcode for the time being to test the sync between both sensors taking an image before the tarball is workin

Reply:
Woohoo! An excellent opportunity to have the opcode take an argument!